.travis.yml

```diff
@@ -23,6 +24,8 @@ before_script:
   - sudo dpkg -i ./vroom_${VROOM_VERSION}-1_all.deb
   - git clone -b ${MAKTABA_VERSION} https://github.com/google/vim-maktaba.git ../maktaba/
   - git clone https://github.com/google/vim-glaive.git ../glaive/
+  - wget https://github.com/kkinnear/zprint/releases/download/${ZPRINT_VERSION}/zprintl-${ZPRINT_VERSION}
```

Comment:
Wait, do you actually need zprint installed for these if you're faking the I/O?

Reply:
I totally did not understand that aspect of vroom. I think the tests will need another pass to straighten this out, but then I think you're right.

Reply:
@dbarnett Rebased now on master with updated tests to fake I/O properly, and removed zprint install from travis.